BloomFilter.md

```diff
@@ -106,15 +116,19 @@ following formula. The output is in bits per distinct element:
 -8 / log(1 - pow(p, 1.0 / 8));
```

Comment:
Is it `c = -8 / log(1 - pow(p, 1.0 / 8))`?

Reply:
The three backticks followed by a string tell Markdown renderers what language to render the fixed-width text as. Here, it is the C language.

Reply:
I was not referring to the c after the three backtick, but to the previous sentence ("The output is in bits per distinct element:") and the definition "Let c := m/n be the bits-per-element rate." in the first linked paper. I think in its current form its unclear what this formula means and if it is indeed c that we calculate it would help to state this by prepending "c = " and adding the defining c. Actually m, n and k would also benefit from having a definitions section.

Reply:
OK, I added some definition in the first section. Please see the latest commit